pages/linux/fwconsole.md

```diff
@@ -0,0 +1,32 @@
+# fwconsole
+
+> Manage and configure your FreePBX system (PBX server).
+> More information: <https://wiki.freepbx.org>.
+
+- Reload FreePBX configurations:
+
+`fwconsole reload`
+
+- Start Asterisk and run other needed FreePBX commands:
+
+`fwconsole start`
+
+- Stop Asterisk and run other needed FreePBX commands:
+
+`fwconsole stop`
+
+- View and update settings:
+
+`fwconsole setting {{keyword}} {{newvalue}}`
```

Comment:
```suggestion
`fwconsole setting {{keyword}} {{new_value}}`
```

Reply:
Hi, I have committed to the changes you suggested. Please let me know if anything else needs to be updated. Thanks for the feedback!